wagtail/admin/rich_text/converters/html_to_contentstate.py

```diff
@@ -215,11 +215,21 @@ def create_entity(self, name, attrs, state, contentstate):
         return Entity('HORIZONTAL_RULE', 'IMMUTABLE', {})
 
 
+class LineBreakHandler:
+    def handle_starttag(self, name, attrs, state, contentstate):
+        state.push()  # void tag
+        state.current_block.text += '\n'
+
+    def handle_endtag(self, name, state, contentstate):
+        state.pop()  # void tag
+
+
 class HtmlToContentStateHandler(HTMLParser):
     def __init__(self, features=()):
         self.paragraph_handler = BlockElementHandler('unstyled')
         self.element_handlers = HTMLRuleset({
-            'p': self.paragraph_handler
+            'p': self.paragraph_handler,
+            'br': LineBreakHandler(),
```

Comment:
If we make this a configurable feature (like your changes to `wagtail_hooks.py` suggest), I don't think this should be in the default element handlers.